lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -28,6 +31,16 @@ def self.get_milestone(repository, release)
         return mile
       end
 
+      # Fetch all the PRs for a given milestone
+      #
+      # @param [String] repository The repository name, including the organization (e.g. `wordpress-mobile/wordpress-ios`)
+      # @param [String] milestone The name of the milestone we want to fetch the list of PRs for (e.g.: `16.9`)
+      # @return [<Sawyer::Resource>] A list of the PRs for the given milestone, sorted by number
+      #
+      def self.get_prs_for_milestone(repository, milestone)
+        github_client().search_issues("type:pr milestone:#{milestone} repo:#{repository}")[:items].sort_by(&:number)
```

Comment:
Quite unlikely to happen in our use cases – except if we want to get the PRs for a frozen milestone with ❄️ in its name 🤔 , but that shouldn't happen in our current process anyway – so not a big deal for our case, but just realized that we should wrap the `milestone` name in quotes in case it contains spaces or special chars.

```suggestion
        github_client().search_issues(%(type:pr milestone:"#{milestone}" repo:#{repository}))[:items].sort_by(&:number)
```